src/NzbDrone.Core/Download/Clients/qBittorrent/QBittorrent.cs

```diff
@@ -116,11 +110,9 @@ public override IEnumerable<DownloadClientItem> GetItems()
                 item.RemainingTime = TimeSpan.FromSeconds(torrent.Eta);
 
                 item.OutputPath = _remotePathMappingService.RemapRemoteToLocal(Settings.Host, new OsPath(torrent.SavePath));
-                
-                // At the moment there isn't an easy way to detect if the torrent has
-                // reached the seeding limit, We would need to check the preferences 
-                // and still not be completely sure if that torrent has a limit set for it
-                item.IsReadOnly = true;
+
+                // Avoid removing torrents that haven't reached their max ratio
+                item.IsReadOnly = config.MaxRatioEnabled && config.MaxRatio > torrent.ShareRatio;
```

Comment:
> This should also check the state of the torrent I think, just to make sure it was actually paused as the user could opt to bypass the ratio check on selected torrents.

sorry, are you saying that a user may want to unpause a completed torrent to prevent Sonarr from importing? i.e.:
```
item.IsReadOnly = torrent.State != "pausedUP" || (config.MaxRatioEnabled && config.MaxRatio > torrent.Ratio);
```
or that a user may want to pause an incomplete torrent so that Sonarr could import it before it reached the ratio limit?
```
item.IsReadOnly = torrent.State != "pausedUP" && (config.MaxRatioEnabled && config.MaxRatio > torrent.Ratio);
```
> I take it the API doesn't yet expose per-torrent ratio limits?

correct, that's not exposed by either the torrent list or the torrent generic properties